mscore/inspector/inspector_line.ui

```diff
@@ -267,7 +267,7 @@
          <double>20.000000000000000</double>
         </property>
         <property name="singleStep">
-         <double>1.000000000000000</double>
+         <double>0.250000000000000</double>
```

Comment:
I guess here old gap width steps are still appropriate?

Reply:
I think you are right. I'll update that.
(I am going to fix dotted slurs and search what else is missing. I think they should be fixed in this PR too).

Reply:
Yes, agreed, slurs (and ties) should use the same